apps/dashboard/lib/trpc/routers/rbac.ts

```diff
@@ -802,6 +803,17 @@ export async function upsertPermissions(
   auditLogs: UnkeyAuditLog[];
 }> {
   return await db.transaction(async (tx) => {
+    // Validate workspace access - allow either user's workspace or system workspace
+    const systemWorkspaceId = env().UNKEY_WORKSPACE_ID;
```

Comment:
is this a good idea?
doesn't this allow people to invite someone into our root workspace if they guess the id?